documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -46,6 +46,18 @@ Limits can be overridden by xref:con-configuring-client-quotas-str[user-specific
 * CPU utilization limits for each client can be set as a percentage of the network threads and I/O threads on a per-broker basis.
 * The number of concurrent partition creation and deletion operations (mutations) allowed per second can be set on a per-broker basis.
 
+.Impact on internal users
+When using the default Kafka quotas plugin, quotas are applied to all users. 
+This includes internal users such as the Topic Operator and Cruise Control, which may impact their operations.
+
+Unlike the Strimzi Quotas plugin, where internal users can be excluded using the `excludedPrincipals` option, the Kafka quotas plugin does not provide an exclusion mechanism.
+To avoid unduly limiting internal users when using the Strimzi Quotas plugin, consider tuning the quotas effectively.
```

Comment:
@PaulRMellor I do not think you should pull the Strmzi Qoutas Pluing in here at all:
* They are not different implementations of the same. They are two different plugins with completely distinct goals and features.
* The Strimzi Qouta Plugin can exclude the users, but:
    * That does not help you here because you use the Kafka plugin if you are reading this
    * It has the internal users excluded automatically, so it has no relevance to the topic.

Maybe this is even more confusing by not using proper chapter structure here? We have `.Quota plugins`, `.strimzi plugin`, `.kafka plugin`, and `.Impact on internal users` which are on the same level.  These should be probably proper section headers with `==` or `===` to make it clear what belongs to what?

Reply:
Right. It's over-complicating it and introducing something we don't need to.
If we include proper headings here, we're separating the content from the steps. We don't have headings inside procedures. So to keep things self-contained and to a minimum,  I've reduced down to just the two plugins and kept this info with the Kafka plugin.